.eslintignore

```diff
@@ -1,2 +1,2 @@
-functions/*.js
-utils/*.js
\ No newline at end of file
+**/*.js
```

Comment:
Is this preventing ESLint from running on all files?

Reply:
Yes, but I believe that's what we want in this project. We'll run ESLint on the `.ts` files.

Reply:
Yes, but in .js files. As this project is TS, we don't care on linting the JS (the code that TS compiler emits is readable but not linting compliant at all).

Reply:
Just what @murilovmachado said